R/biome.R

```diff
@@ -69,10 +74,12 @@ split_biome <- function(core,
         length(fveg_c) == length(new_biomes),
         length(fdetritus_c) == length(new_biomes),
         length(fsoil_c) == length(new_biomes),
+        length(fpermafrost_c) == length(new_biomes),
         length(fnpp_flux0) == length(new_biomes),
         sum(fveg_c) == 1, all(fveg_c > 0),
         sum(fdetritus_c) == 1, all(fdetritus_c > 0),
         sum(fsoil_c) == 1, all(fsoil_c > 0),
+        sum(fpermafrost_c) == 1, all(fpermafrost_c > 0),
```

Comment:
this should be all(fpermafrost_c >= 0) since many biomes will not have permafrost in them

Reply:
Good catch—thanks.